Sources/docc/DocCDocumentation.docc/formatting-your-documentation-content.md

```diff
@@ -1,6 +1,6 @@
 # Formatting Your Documentation Content
 
-Enhance your content's presentation with special formatting and styling for text, links, and other page elements.
+Enhance your content's presentation with special formatting and styling for text, and lists.
```

Comment:
Maybe remove the comma on this line: "text and lists."

Reply:
Thx for the suggestion Pat, resolved in https://github.com/apple/swift-docc/pull/742/commits/1d13acc804cd78a503e3a9ef31e6a79f2f72915e